qiskit_experiments/calibration/experiments/rabi.py

```diff
@@ -303,3 +303,92 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             circs.append(assigned_circ)
 
         return circs
+
+
+class EFRabi(Rabi):
+    """An experiment that scans the amplitude of a pulse to calibrate rotations between 1 and 2.
+
+    This experiment is similar to the Rabi experiment between the ground and first excited state.
+    The difference is in the initial X gate used to populate the first excited state. The Rabi pulse
+    is then applied on the 1 <-> 2 transition (sometimes also labeled the e <-> f transition) which
+    implies that frequency shift instructions are used. The necessary frequency shift (typically the
+    qubit anharmonicity) should be specified through the experiment options.
+
+    The circuits are of the form:
+
+    .. parsed-literal::
+
+                   ┌───┐┌───────────┐ ░ ┌─┐
+              q_0: ┤ X ├┤ Rabi(amp) ├─░─┤M├
+                   └───┘└───────────┘ ░ └╥┘
+        measure: 1/══════════════════════╩═
+                                         0
+    """
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default values for the pulse if no schedule is given.
+
+        Users can set a schedule by doing
+
+        .. code-block::
+
+            ef_rabi.set_experiment_options(schedule=rabi_schedule)
+
+        """
+        return Options(
+            duration=160,
+            sigma=40,
+            amplitudes=np.linspace(-0.95, 0.95, 51),
+            schedule=None,
+            normalization=True,
+            frequency_shift=None,
+        )
+
+    def _default_gate_schedule(self, backend: Optional[Backend] = None):
+        """Create the default schedule for the EFRabi gate with a frequency shift to the 1-2
+        transition."""
+
+        if self.experiment_options.frequency_shift is None:
+            try:
+                anharm, _ = backend.properties.qubit_property(self.physical_qubits[0])[
+                    "anharmonicity"
+                ]
+                self.set_experiment_options(frequency_shift=anharm)
+            except KeyError as key_err:
+                raise QiskitError(
+                    f"The backend {backend} does not provide an anharmonicity for qubit "
+                    f"{self.physical_qubits[0]}. Use EFRabi.set_experiment_options(frequency_shift="
+                    f"anharmonicity) to manually set the correct frequency for the 1-2 transition."
+                ) from key_err
+            except AttributeError as att_err:
+                raise QiskitError(
```

Comment:
```suggestion
                raise CalibrationError(
```
Note that you will need to import this. Also I suggest waiting for #166 to be in first so `CalibrationError` will be in `qiskit_experiments.exceptions`

Reply:
Thanks Daniel, addressed in 65443e2.